spec/models/game_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+RSpec.describe Game, type: :model do
+  let(:game) { create(:game_with_matches) }
+
+  describe ".name" do
+    it "is a string" do
+      expect(game.name).to be_a(String)
+    end
+  end
+
+  describe ".image" do
+    it "is present" do
+      expect(game.image).to be_present
+    end
+  end
+
+  describe "associations" do
+    it "includes matches" do
+      expect(game.match).not_to be_empty
```

Comment:
I think this test does not represent behaviour of the Game as a unit thus it shouldn't be here